path_config_test.go

```diff
@@ -44,46 +44,88 @@ func setupLocalFiles(t *testing.T, b logical.Backend) func() {
+				"kubernetes_host":        "host",
+				"kubernetes_ca_cert":     testCACert,
+				"issuer":                 "",
+				"disable_iss_validation": false,
+				"disable_local_ca_jwt":   false,
+			},
+			want: map[string]interface{}{
+				"pem_keys":               []string{testRSACert, testECCert},
+				"kubernetes_host":        "host",
+				"kubernetes_ca_cert":     testCACert,
+				"issuer":                 "",
+				"disable_iss_validation": false,
+				"disable_local_ca_jwt":   false,
+				"token_reviewer_jwt_set": false,
+			},
+		},
+		{
+			name: "token-review-jwt-is-set",
+			data: map[string]interface{}{
+				"pem_keys":               []string{testRSACert, testECCert},
+				"kubernetes_host":        "host",
+				"kubernetes_ca_cert":     testCACert,
+				"issuer":                 "",
+				"disable_iss_validation": false,
+				"disable_local_ca_jwt":   false,
+				"token_reviewer_jwt":     "test-token-review-jwt",
+			},
+			want: map[string]interface{}{
+				"pem_keys":               []string{testRSACert, testECCert},
+				"kubernetes_host":        "host",
+				"kubernetes_ca_cert":     testCACert,
+				"issuer":                 "",
+				"disable_iss_validation": false,
+				"disable_local_ca_jwt":   false,
+				"token_reviewer_jwt_set": true,
+			},
+		},
 	}
 
-	req := &logical.Request{
-		Operation: logical.UpdateOperation,
-		Path:      configPath,
-		Storage:   storage,
-		Data:      data,
-	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			b, storage := getBackend(t)
+			cleanup := setupLocalFiles(t, b)
+			t.Cleanup(cleanup)
 
-	resp, err := b.HandleRequest(context.Background(), req)
-	if err != nil || (resp != nil && resp.IsError()) {
-		t.Fatalf("err:%s resp:%#v\n", err, resp)
-	}
+			req := &logical.Request{
+				Operation: logical.UpdateOperation,
+				Path:      configPath,
+				Storage:   storage,
+				Data:      tc.data,
+			}
 
-	req = &logical.Request{
-		Operation: logical.ReadOperation,
-		Path:      configPath,
-		Storage:   storage,
-		Data:      nil,
-	}
+			resp, err := b.HandleRequest(context.Background(), req)
+			if err != nil || (resp != nil && resp.IsError()) {
+				t.Fatalf("err:%s resp:%#v\n", err, resp)
+			}
 
-	resp, err = b.HandleRequest(context.Background(), req)
-	if err != nil || (resp != nil && resp.IsError()) {
-		t.Fatalf("err:%s resp:%#v\n", err, resp)
-	}
+			req = &logical.Request{
+				Operation: logical.ReadOperation,
+				Path:      configPath,
+				Storage:   storage,
+				Data:      nil,
+			}
 
-	if !reflect.DeepEqual(resp.Data, data) {
-		t.Fatalf("Expected did not equal actual: expected %#v\n got %#v\n", data, resp.Data)
+			resp, err = b.HandleRequest(context.Background(), req)
+			if err != nil || (resp != nil && resp.IsError()) {
+				t.Fatalf("err:%s resp:%#v\n", err, resp)
+			}
+
+			if !reflect.DeepEqual(resp.Data, tc.want) {
+				t.Fatalf("Expected did not equal actual: expected %#v\n got %#v\n", tc.want, resp.Data)
```

Comment:
The message seems to infer that we were for testing for equality, so I think we can make it slightly less verbose. Also the extra newlines tend to make reading the results a bit more difficult, so we should probably strip those out.
```suggestion
				t.Fatalf("expected %#v, got %#v", tc.want, resp.Data)
```